core/src/utils/overlays.ts

```diff
@@ -385,6 +387,17 @@ export const getOverlays = (doc: Document, selector?: string): HTMLIonOverlayEle
   return (Array.from(doc.querySelectorAll(selector)) as HTMLIonOverlayElement[]).filter((c) => c.overlayIndex > 0);
 };
 
+/**
+ * Returns a list of all presented overlays.
+ * Inline overlays can exist in the DOM but not be presented,
+ * so there are times when we want to exclude those.
+ * @param doc The document to find the element within.
+ * @param overlayTag The selector for the overlay, defaults to Ionic overlay components.
+ */
+const getPresentedOverlays = (doc: Document, overlayTag?: string): HTMLIonOverlayElement[] => {
+  return getOverlays(doc, overlayTag).filter((o) => !isOverlayHidden(o));
+};
+
 /**
  * Returns an overlay element
```

Comment:
Should we either update the comment on this function to mention it returns a presented overlay element and/or refactor the function name to `getPresentedOverlay`? 

I can see the current naming overlap being confusing:
- `getOverlays` - returns visible/non-visible overlays
- `getPresentedOverlays` - returns only visible overlays
- `getOverlay` - returns only a visible overlay

Reply:
Good thought. Clarified in https://github.com/ionic-team/ionic-framework/pull/28183/commits/905bd8662da7001c5f05fb1771d4ffaab81bd3bb and https://github.com/ionic-team/ionic-framework/pull/28183/commits/3e7b65ddd057933810aff489770c7f1bec20179a